ds000117/sub-01/ses-meg/meg/sub-01_ses-meg_task-facerecognition_run-01_events.tsv

```diff
@@ -1,147 +1,147 @@
 onset	duration	onset_sample	stim_type	trigger	stim_file
-26.078	0	26628	Unfamiliar	13	meg/u032.bmp
```

Comment:
@robertoostenveld Sorry, we had been discussing this over email since not everyone is on github. I'll foward it to you.